reconcile/change_owners/change_owners.py

```diff
@@ -319,20 +335,23 @@ def run(
         #   C H A N G E   C O V E R A G E
         #
         changes = fetch_bundle_changes(comparison_sha)
+        logging.info(
+            f"detected {len(changes)} changed bundle files with {sum(len(c.diff_coverage) for c in changes)} differences"
```

Comment:
nit: i'd remove `bundle` in this log message to avoid thinking "bundle files" are changed (there would be only 1?)

Reply:
good point. done in [099b0f4](https://github.com/app-sre/qontract-reconcile/pull/3052/commits/099b0f41286acf5fb843209c1e74b3ae730cac34)